### PR TITLE
create hello-microverse project and test linter error

### PR DESCRIPTION
**Pull request by MariamOsman11**
# What I have done
`configure linters`
-Create branches with different base branches.
-create html and css files
 -Open pull requests in GitHub.
-Write descriptive, easy to understand commit messages.
-Write short, detailed, easy to understand descriptions of pull requests.
-Write a descriptive and easy to understand README in business English for every project submitted.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,64 @@
...
+code editor
+web browser
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**Marian Abdihakin Isman
+
+- GitHub: [@githubhandle](https://github.com/MariamOsman)
```
Comment (GedeonTS) [inline README.md:39]: __Kindly remember to always submit the same Pull request fo re-review to allow the track of improvement on the milestone__
- [ ] Good use of the Microverse template, Kindly make sure to add all missing sections assuming the only optional is FAQ :recycle: to make your project more professional :recycle:

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @MariamOsman11 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- No linter errors :+1: 
- Good use of GitHub flow :+1: 
- Good use of gitignore :+1: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
- [OPTIONAL]  I think it would be better to always start your commit messages with a capital letter to make your work look more professional :recycle: 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MariamOsman11** [comment]
# what I have done
-update readme file

**Review by tugsem — APPROVED** [review]
## STATUS: APPROVED 👍🎯  
Hi @MariamOsman11 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⭐ 
✔️ Readme is done well

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**